### PR TITLE
Add Lightning Test Create command

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?
This PR adds the `sfdx force:lightning:test:create` command

### What issues does this PR fix or reference?
@W-6783783@

### CODE REVIEW

**Comment by AnanyaJha** [comment]
Tests still need to be added

File: test/commands/lightning/test/create.test.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+ */
+import { expect, test } from '@salesforce/command/lib/test';
+import { Messages } from '@salesforce/core';
+import * as path from 'path';
+import * as assert from 'yeoman-assert';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('salesforcedx-templates', 'messages');
+
+describe('Lightning test creation tests:', () => {
```
Comment (sfsholden) [inline test/commands/lightning/test/create.test.ts]: I think the original description 'Lightning test creation suite' reads better here

File: test/commands/lightning/test/create.test.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      .stdout()
+      .command([
+        'force:lightning:test:create',
+        '-n',
+        'foo',
+        '--template',
+        'DefaultLightningTest'
+      ])
+      .it(
+        'should create lightning test foo using DefaultLightningTest template and aura output directory',
```
Comment (sfsholden) [inline test/commands/lightning/test/create.test.ts]: This test isn't generating the aura output directory like it's described here

File: test/commands/lightning/test/create.test.ts (hunk 1 above)
Comment (lcampos) [inline test/commands/lightning/test/create.test.ts]: We should have a test that confirm the `--internal` flag scenario.

Comment (AnanyaJha) [reply test/commands/lightning/test/create.test.ts]: I agree the double 'test' sounds a little odd, but I left it as is to follow the naming convention used across the other tests in this repo

File: test/commands/lightning/test/create.test.ts (hunk 2 above)
Comment (AnanyaJha) [reply test/commands/lightning/test/create.test.ts]: Ah good catch

**Pull request merged**